index.html

```diff
@@ -132,5 +123,251 @@
+                        </div>
+                    </div>
+                    <h2 class="project-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h2>
+                    <ul class="tags">
+                        <li class="tags-list"><a class="tags-text" href="">html</a></li>
+                        <li class="tags-list"><a class="tags-text" href="">css</a></li>
+                        <li class="tags-list"><a class="tags-text" href="">javaScript</a></li>
+                    </ul>
+                    <div class="actions">
+                        <button>
+                            <a href="">See&nbspProject</a>
+                        </button>
+                    </div>
+                </div>
+        </div>
+        <div class="card-works">
+                <div class="work-snapshot-4">
+                </div>
+                <div class="work-block">
+                    <div class="primary-text">
+                        <h1 class="project-title">Tonic</h1>
+                        <div class="frame">
+                            <div class="client"><p>CANOPY</p></div>
+                            <div>
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                </svg>
+                            </div>
+                            <div class="role"><p>Back&nbspEnd&nbspDev</p></div>
+                            <div>
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                </svg>
+                            </div>
+                            <div class="year"><p>2015</p></div>
+                        </div>
+                    </div>
+                    <h2 class="project-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h2>
+                    <ul class="tags">
+                        <li class="tags-list"><a class="tags-text" href="">html</a></li>
+                        <li class="tags-list"><a class="tags-text" href="">css</a></li>
+                        <li class="tags-list"><a class="tags-text" href="">javaScript</a></li>
+                    </ul>
+                    <div class="actions">
+                        <button>
+                            <a href="">See&nbspProject</a>
+                        </button>
+                    </div>
+                </div>
+        </div>
+    </main>
+
+    <section class="about-myself">
+        <div class="top">
+            <h1>
+                About <br>
+                Myself
+            </h1>
```

Comment:
- [ Required ] A well-designed webpage usually has one `h1` tag which marks the most important title on the webpage, your webpage appears to have more than one which is not good practice. Please consider changing this `h1` tag to a `h2` tag to make your webpage semantically correct.